index.js

```diff
@@ -73,37 +77,37 @@
       return tip
     }
 
-    // Public: Proxy attr calls to the d3 tip container.  Sets or gets attribute value.
+    // Public: Proxy attr calls to the d3 tip container.
+    // Sets or gets attribute value.
     //
     // n - name of the attribute
     // v - value of the attribute
     //
     // Returns tip or attribute value
-    tip.attr = function(n, v) {
+    tip.attr = function(n, _v) {
       if (arguments.length < 2 && typeof n === 'string') {
         return getNodeEl().attr(n)
-      } else {
-        var args =  Array.prototype.slice.call(arguments)
-        d3.selection.prototype.attr.apply(getNodeEl(), args)
       }
 
+      var args =  Array.prototype.slice.call(arguments)
+      d3.selection.prototype.attr.apply(getNodeEl(), args)
       return tip
     }
 
-    // Public: Proxy style calls to the d3 tip container.  Sets or gets a style value.
+    // Public: Proxy style calls to the d3 tip container.
+    // Sets or gets a style value.
     //
     // n - name of the property
     // v - value of the property
     //
     // Returns tip or style property value
-    tip.style = function(n, v) {
+    tip.style = function(n, _v) {
```

Comment:
That was a custom setting I added to [`no-unused-vars`](http://eslint.org/docs/rules/no-unused-vars) to ignore unused variables matching a certain pattern. It's used in the rule examples but AFAIK it's not standardised anywhere, so I understand if you'd prefer to remove it. Some alternatives would be:

1. Add `// eslint-ignore-next-line no-unused-vars` above the two methods to exempt them from the rule, or
2. Configure `no-unused-vars` with the `args: none` option to skip checks on all arguments.